sam/slack.py

```diff
@@ -231,6 +232,43 @@ def fetch_coworker_contacts(_context=None) -> str:
         return json.dumps(profiles)
 
 
+def fetch_slack_history(latest_ts: str, oldest_ts: str, _context=None) -> str:
+    """Fetch messages from a Slack channel.
+
+    Args:
+        latest_ts: The latest unix timestamp of the messages to fetch.
+        oldest_ts: The oldest unix timestamp of the messages to fetch.
```

Comment:
I find this misleading. It's a range, it will get all messages within the range. However, the timestamps don't have to actually match a specific message timestamp.